examples/SampleUtils.h

```diff
@@ -120,45 +120,16 @@ namespace utils {
         const ml::Operand& operand;
     } NamedOperand;
 
-    ml::Graph AwaitBuild(const ml::GraphBuilder& builder, const std::vector<NamedOperand>& outputs);
+    ml::Graph Build(const ml::GraphBuilder& builder, const std::vector<NamedOperand>& outputs);
 
     typedef struct {
         const std::string name;
-        const ml::Input input;
-    } NamedInput;
+        const std::vector<float>& resource;
+    } NamedResource;
```

Comment:
Could it be a template struct? e.g., `std::vector<T>& resource`

Reply:
All tests use const data for input, for example

https://github.com/webmachinelearning/webnn-native/blob/main/src/tests/end2end/AddTests.cpp#L37
`const std::vector<float> dataA =`

should we define a struct for output?
```
typedef struct {
        const std::string name;
        const std::vector<float>& resource;
    } NamedInput;

typedef struct {
        const std::string name;
        std::vector<float>& resource;
    } NamedOutput;
```

Reply:
sounds good, thanks!